prov/verbs/src/ep_rdm/verbs_tagged_ep_rdm.c

```diff
@@ -544,20 +544,21 @@ fi_ibv_rdm_tagged_process_recv(struct fi_ibv_rdm_ep *ep,
 		}
 	}
 
-	struct fi_ibv_recv_got_pkt_preprocess_data p = {
-		.conn = conn,
-		.ep = ep,
-		.rbuf = rbuf,
-		.arrived_len = arrived_len,
-		.pkt_type = pkt_type,
-		.imm_data = imm_data
-	};
+	/* RMA packets are not handled yet (without IMM) */
+	if (pkt_type != FI_IBV_RDM_RMA_PKT) {
```

Comment:
No, then FI_IBV_RDM_RNDV_ACK_PKT will be lost. This condition is always true now.